ElrondNetwork/api.elrond.com
# Gateway updates

## Type

## Problem setting
- add support for new gateway functionality
  
## Proposed Changes
- extended supply info for tokens
- extract peerType from heartbeat first, then from validator statistics

## How to test (mainnet)
- `/tokens/MEX-455c57` should contain `minted`, `burnt`, `initialMinted` fields
- `/tokens/MEX-455c57/supply` should contain `minted`, `burnt`, `initialMinted` fields
- `/nodes/count?type=validator` should return 3200 + queue size (at the moment of PR open, it is 3269)
